packages/expo-notifications/src/NotificationCategoriesModule.ts

```diff
@@ -11,7 +11,7 @@ export default {
   ): Promise<null> {
```

Comment:
Also add the correct types to `options` arg for this since this is now the file used to dictate the types of the methods. Can also use `NotificationCategoryOptions` in `NotificationCategoriesModule.types.ts`
```typescript
options?: NotificationCategoryOptions
```